### DISCUSSION TITLE
Configuration hot reload for Spring boot?

### DISCUSSION THREAD

**Discussion by putra4648** [mfussenegger/nvim-jdtls#373] [Q&A] [answered]
How to configure auto reload feature for spring boot? 
Here I use thymeleaf and spring-boot devtools, after making changes to the files in the html it doesn't change immediately and I have to do a hard restart like killing the current process and restarting it.
Where is the error?

**Comment by EpsilonKu** [comment, marked as answer]
You need put this to `application.properties`:
```
devtools.livereload.enabled = true
spring.thymeleaf.prefix=file:src/main/resources/templates/
spring.thymeleaf.cache=false
```
Also install `https://mvnrepository.com/artifact/org.springframework.boot/spring-boot-devtools`.
So it will automatically hot-reload from spring side. It doesn't require jdtls hot reload.

**Reply by EpsilonKu** [reply]
It also good with any IDE. Works really good.

**Reply by putra4648** [reply]
Thank you! Finally everything worked fine
